CMakeLists.txt

```diff
@@ -170,7 +171,6 @@ if (error EQUAL 0)
   ADD_SUBDIRECTORY(src/lib/xmlParse)
   ADD_SUBDIRECTORY(src/lib/rest)
   ADD_SUBDIRECTORY(src/lib/mongoBackend)
-  ADD_SUBDIRECTORY(src/lib/cache)
```

Comment:
Note that the library 'cache' is removed from compilation and linking, but the files are still in git.
Once the sub-cache is fully accepted, we'll remove the files under lib/cache as well.